### ISSUE TITLE
`corp-semantic-release` is forgetting to commit lock files

### ISSUE THREAD

**Issue by rodoabad** [leonardoanalista/corp-semantic-release#64]
I have noticed that `corp-semantic-release` is not committing and pushing lock files. Is anyone else having this problem?

**Comment by leonardoanalista** [comment]
yes, you are right. This is the line which adds the files: https://github.com/leonardoanalista/corp-semantic-release/blob/master/src/lib/addFilesAndCreateTag.js#L10

If you have a chance to right a PR let me know. Otherwise give me a few hours to be able to write a PR.

**Comment by leonardoanalista** [comment]
That line could potentially be `code = shell.exec('git add package.json package-lock.json CHANGELOG.md').code;`

Code should add and commit `package-lock.json` as part of the changes. This is a possible breaking changes as some could be using `yarn.lock` instead. 

Alternatively, a preferred solution would be identify if the project uses `npm` or `yarn.lock` and add/commit the appropriated file.

**Comment by rodoabad** [comment]
@leonardoanalista let's do a little bit of hashing out the details. You've identified one challenge and that is to try and identify if the consumer is using either `npm` or `yarn` or it could even be both (they update both lock files).

I think one more challenge is to identify if they have set `npm` and `yarn` to not generate lock files which is what I did for the mean time in order to not trigger two consecutive CI runs.

**Comment by rodoabad** [comment]
@leonardoanalista here's a rough snippet. I can submit a PR if you want too. Is this what you had in mind?

```js
  code = shell.exec('git add package.json package-lock.json CHANGELOG.md').code;
  if (shell.test('-f', 'package-lock.json')) {
    code = shell.exec('git package-lock.json').code;
  }
  if (shell.test('-f', 'yarn.lock')) {
    code = shell.exec('git yarn.lock').code;
  }
```

**Comment by leonardoanalista** [comment]
Maybe the first line can do what it was doing before:

`code = shell.exec('git add package.json CHANGELOG.md').code;`

Then have the 2 `if` conditions to add package or yarn lock files if they exist.

**Comment by rodoabad** [comment]
LOL. Good catch.

**Comment by BBlackwo** [comment]
`yarn.lock` doesn't contain the package version so it shouldn't need to be committed, right? Or am I missing something?

**Comment by rodoabad** [comment]
@leonardoanalista do you have time to do this? I am getting pulled away from things right now. I"m sorry!

**Comment by leonardoanalista** [comment]
Bblackwo made a good point. I m on holidays now and can’t look into details but I agree with him.
We need to see how we use this package on the apps. When we run a release, this package modified package.json, add and commit. I ll need to look at this when I return

**Comment by rodoabad** [comment]
NP. Thanks, @BBlackwo!

**Comment by rodoabad** [comment]
@leonardoanalista any update on this?
